customizations/README.md

```diff
@@ -344,9 +346,13 @@ REDIS_TLS_ENABLED=true
 
 #### Configuration
 
-CARTO self-hosted provides support for operating behind an **HTTP** proxy. The proxy acts as an HTTP gateway, enabling CARTO self-hosted components to establish connections with essential external services like Google APIs, Mapbox, and others.
+CARTO self-hosted provides support for operating behind an HTTP or HTTPS proxy. The proxy acts as a gateway, enabling CARTO self-hosted components to establish connections with essential external services like Google APIs, Mapbox, and others.
 
-In order to enable this feature, set the following environment variables (both uppercase and lowercase variables) in your `.env` file:
+A comprehensive list of domains that must be whitelisted by the proxy for the proper functioning of CARTO self-hosted can be found [here](../customizations/proxy/config/whitelisted_domains). The list includes domains for the essential core services of CARTO self-hosted, as well as additional optional domains that should be enabled to access specific features.
+
+In order to enable this feature, set the following environment variables (both uppercase and lowercase variables) in your `.env` file, depending on the protocol your proxy uses.
```

Comment:
```suggestion
In order to enable this feature, set the following environment variables (both uppercase and lowercase variables) in your `.env` file.
```

I think it was a bit misleading, because you need to setup all of them, no matter which protocol is used in the proxy.